oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/dsl/SampleFamily.java

```diff
@@ -464,7 +464,7 @@ public SampleFamily instance(List<String> serviceKeys, List<String> instanceKeys
         return instance(serviceKeys, Const.POINT, instanceKeys, Const.POINT, layer, null);
     }
 
-    public SampleFamily endpoint(List<String> serviceKeys, List<String> endpointKeys, Layer layer) {
+    public SampleFamily endpoint(List<String> serviceKeys, List<String> endpointKeys, String delimiter, Layer layer) {
```

Comment:
Could you explain a little more about this new delimiter parameter? What happens?

Reply:
As mentioned above,  `router` is made by `Method(route method)` and `Resource(route path)` such as `GET /students`.
`endpoint(['service_name'],['Method','Resource'], Layer.AWS_GATEWAY)`  will generate an endpoint that contains a dot by default, such as `GET./students`. It's better to use space to replace dot to display.

Alternative, ` tags.endpoint_name= tags.Method+' '+tags.Resource` also could generate an endpoint with a space delimiter.